src/utils/axiosWithAuth.js

```diff
@@ -1,12 +1,15 @@
 import axios from 'axios';
 
-const axiosWithAuth = idToken => {
-  return axios.create({
-    baseURL: process.env.REACT_APP_API_URI,
-    headers: {
-      Authorization: 'Bearer ' + idToken,
-    },
-  });
-};
+//TO-DO: Implement Auth0
+const axiosWithAuth = "Implement Auth0";
```

Comment:
I would be careful here because any reference to `axiosWithAuth` that's expecting a promise will now be receiving a string. My recommendation would be to leave the function as it is and only comment out the headers props.

Reply:
That makes sense